Composer/packages/extensions/obiformeditor/src/Form/widgets/LuEditorWidget.tsx

```diff
@@ -78,11 +78,19 @@ export class LuEditorWidget extends React.Component<LuEditorWidgetProps> {
   render() {
     const { height = 250 } = this.props;
     const { luFile, luFileId, luIntent, name } = this;
-    const diagnostic = luFile && filterSectionDiagnostics(luFile.diagnostics, luIntent)[0];
 
-    const errorMsg = diagnostic
-      ? diagnostic.message.split('error message: ')[diagnostic.message.split('error message: ').length - 1]
-      : '';
+    const combineMessage = (diagnostics: Diagnostic[]): string => {
```

Comment:
ditto. 1. LU & LG are too similar to be separated components. 2. dialogstic logic might belong to LU/LG editor lib

Reply:
The CRUD/parse logic of lg/lu are not  100% same. Abstract them into one component would bring more maintenance cost, so  keep two standalone components at this moment is more flexiable. except the copied code are not that looks fancy.